src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptOutOasis.java

```diff
@@ -60,10 +57,10 @@ public void adsTrackingOptOutOasis() {
   }
 
   public static String appendTrackingOptOutParameters(String url) {
-    UrlBuilder urlBuilder = new UrlBuilder();
+    UrlBuilder urlBuilder = UrlBuilder.createUrlBuilder();
 
     for (String instantGlobal : INSTANT_GLOBALS) {
-      url = urlBuilder.globallyEnableGeoInstantGlobalOnPage(url, instantGlobal);
+      url = UrlBuilder.createUrlBuilder().globallyEnableGeoInstantGlobalOnPage(url, instantGlobal);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Introduce a new variable instead of reusing the parameter "url". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)